designcompose/src/main/java/com/android/designcompose/DesignView.kt

```diff
@@ -936,6 +940,7 @@ internal fun DesignDocInternal(
     parentComponents: List<ParentComponentInfo> = listOf(),
     parentLayout: ParentLayoutInfo? = null,
 ) {
+    beginSection(DCTraces.DESIGNDOCINTERNAL)
```

Comment:
Where's the `endSection()` for this, and other `beginSection()` calls that don't have an `endSection()`?

Reply:
Good catch, looks like I lost the endSection for this.